### ISSUE TITLE
Wrong json for 'show jobs'

### ISSUE THREAD

**Issue by IvanBayan** [bareos/python-bareos#8]
> In [14]: directorconsole.call('show jobs')
> Out[14]: 
> {'error': {'code': 2,
>   'data': bytearray(b'Job {\n  Name = "RestoreFiles"\n  Description = "Standard Restore template. Only one such job is needed for all standard Jobs/Clients/Storage ..."\n  Type = Restore\n  Messages = "Standard"\n  Storage = "mia-backup03_storage03-sd"\n  Pool = "mia-backup03_mixed_incremental"\n  Client = "mia-backup03-fd"\n  FileSet = "LinuxAll"\n  Where = "/tmp/bareos-restores"\n}\n\n ##CUT##}\n\n{"result":{},"jsonrpc":"2.0","id":null}'),
>   'message': 'No JSON object could be decoded'}}

```
% dpkg -l|grep python-bareos
ii  python-bareos                             0.4.1509724938.4a5bd79-48.1                 all          Backup Archiving REcovery Open Sourced - Python module

```

**Comment by joergsteffens** [comment]
Yes, this is a known limitation. Not all Bareos Director commands support the JSON format. Only the most used once (or used by the bareos-webui) are implemented. ``show`` is not and ``status`` only contains part of the information. There may be the chance, that ``show`` is implemented in the future, but you are the first one who pointed this out.
